airflow/helm/airflow/values.yaml

```diff
@@ -27,8 +27,8 @@ postgres:
 
 exporter:
   image:
-    repository: gcr.io/pluralsh/prom/statsd-exporter
-    tag: v0.20.2
```

Comment:
the tag should be `v0.23.0` based on the image vendoring.

Reply:
it's not in our registry yet I think